tests/scaffoldings/dotd/custom/entrypoint.d/600-goodconf.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/local/bin/python-odoo-shell
+from odoo.tools import config
```

Comment:
There's a problem on putting these test files at the entrypoint: both get executed before each test in `dotd`, and there are a lot.

Put this file in a separate script `dotd/custom/extras/goodconf.py`. Then [add a command to the list](https://github.com/Tecnativa/docker-odoo-base/blob/master/tests/__init__.py#L194) that executes this script. Make sure you add the command after another test that should have created a database, this way you don't have to worry about the DB creation thing.